### PR TITLE
add r-multbxxc

### PR DESCRIPTION
**Pull request by sgsokol**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by sgsokol** [comment]
@conda-forge/help-r please review

File: recipes/r-multbxxc/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/multbxxc_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/multbxxc/multbxxc_{{ version }}.tar.gz
+  sha256: f5911f334d3390cbb7747fa6db95a7c792e74117c493acadf15752c12457e5c6
+
+build:
+  merge_build_host: True  # [win]
+  # If this is a new build for the same version, increment the build number.
```
Comment (npavlovikj) [inline recipes/r-multbxxc/meta.yaml]: Please remove comment.

File: recipes/r-multbxxc/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+  number: 0
+
```
Comment (npavlovikj) [inline recipes/r-multbxxc/meta.yaml]: Please remove new line.

File: recipes/r-multbxxc/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+  # This is required to make R link correctly on Linux.
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+# Suggests: testthat
```
Comment (npavlovikj) [inline recipes/r-multbxxc/meta.yaml]: Remove comment.

File: recipes/r-multbxxc/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{posix}}filesystem        # [win]
+    - {{posix}}make
+    - {{posix}}sed               # [win]
+    - {{posix}}coreutils         # [win]
+    - {{posix}}zip               # [win]
+
```
Comment (npavlovikj) [inline recipes/r-multbxxc/meta.yaml]: Remove new line.

File: recipes/r-multbxxc/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+  host:
+    - r-base
+    - r-rcpp >=1.0.0
+    - r-rcpparmadillo
+    - r-rmumps >=5.2.1_6
+    - libblas
+    - liblapack
+
```
Comment (npavlovikj) [inline recipes/r-multbxxc/meta.yaml]: Remove new line.

File: recipes/r-multbxxc/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+    - {{native}}gcc-libs         # [win]
+    - r-rcpp >=1.0.0
+    - r-rcpparmadillo
+    - r-rmumps >=5.2.1_6
+    - libblas
+    - liblapack
+
+test:
+  commands:
+    # You can put additional test commands to be run here.
```
Comment (npavlovikj) [inline recipes/r-multbxxc/meta.yaml]: Remove comment.

File: recipes/r-multbxxc/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+    - $R -e "library('multbxxc')"           # [not win]
+    - "\"%R%\" -e \"library('multbxxc')\""  # [win]
+
+  # You can also put a file called run_test.py, run_test.sh, or run_test.bat
```
Comment (npavlovikj) [inline recipes/r-multbxxc/meta.yaml]: Remove all these comments.

File: recipes/r-multbxxc/meta.yaml [hunk 8]
```diff
@@ -0,0 +1,106 @@
...
+
+  # requires:
+    # Put any additional test requirements here.
+
+about:
+  home: https://metasys.insa-toulouse.fr/software/influx/
+  license: GPL (>= 2)
+  summary: 'Contains auxiliary routines for influx software. This packages is not intended to
+    be used directly. Influx was published here: Sokol et al. (2012) <doi:10.1093/bioinformatics/btr716>.'
+
```
Comment (npavlovikj) [inline recipes/r-multbxxc/meta.yaml]: Remove line.

File: recipes/r-multbxxc/meta.yaml [hunk 9]
```diff
@@ -0,0 +1,106 @@
...
+  license_family: GPL3
```
Comment (npavlovikj) [inline recipes/r-multbxxc/meta.yaml]: Use `GPL` instead of `GPL3`.

**Comment by npavlovikj** [comment]
In general, the recipe looks ok. Please just clean it as suggested before we go ahead and merge it.

File: recipes/r-multbxxc/meta.yaml (hunk 9 above)
Comment (sgsokol) [reply recipes/r-multbxxc/meta.yaml]: Cleaning up is done. Thanks for your ant work.

Comment (npavlovikj) [reply recipes/r-multbxxc/meta.yaml]: No problem @sgsokol  - thank you for contributing to CF as always :)

**Pull request merged**